sphinx/source/docs/user_guide/source_examples/extensions_example_latex.py

```diff
@@ -13,7 +13,8 @@
 class LatexLabelView extends Label.View
   render: () ->
 
-    ### Copied directly from the ``Label.render`` implementation
+    # Start of copied section from ``Label.render`` implementation
+    
```

Comment:
CoffeeScript is such a badly designed (if at all) language. Why would the author bother to appreciate historical patterns in code writing. Anyway, soon we will be able to use external files for custom models' implementations, so one will be able to use code highlighting to avoid such mistakes.